msresist/figures/figureS5.py

```diff
@@ -0,0 +1,76 @@
+"""
+This creates Supplemental Figure 3: Prediction performance across different plsr models using different clustering strategies
```

Comment:
I think this information should be in the main figure and we don't need a supplementary one.

- The performance versus component number can go—readers don't need to see this information if you're properly picking the right number of components.
- All the different methods can be presented in one plot using a stripplot like this: 

![](https://seaborn.pydata.org/_images/seaborn-stripplot-6.png)